src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/URLValidation.feature.disabled

```diff
@@ -0,0 +1,385 @@
+Feature:
+
+        @ignore
+        Scenario: 01 - Ensure clients cannot retrieve information if part of the endpoint is missing
+             When a GET request is made to "/schools"
```

Comment:
I think @stephenfuqua added a validation when the url is missing the `data/edfi` part. This could be found in the method called `addDataPrefixIfNecessary`, when the test got enabled we need to make sure it works as expected.

Reply:
It would be good to see if this can be enabled. Before merging this pull request, try enabling each scenario 1 at a time to see (a) does it compile without error and (b) does the test pass?

Reply:
Project compiles but the tests are failing.  Tickets are going to be created to fix those errors, because one of them is an error, and the other one is a bug.